keyboards/tetris/config.h

```diff
@@ -36,22 +32,30 @@
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
-#define NO_ACTION_MACRO
-#define NO_ACTION_FUNCTION
+/* key combination for command */
+#define IS_COMMAND() ( \
```

Comment:
Please remove this, as it's the default, and already set.

Additionally, please re-add the `NO_ACTION` defines.